tests/test_inclined_exponential.py

```diff
@@ -126,211 +281,223 @@ def test_edge_on():
-
-        # Get float values for the details
-        flux = float(flux)
-        inc_angle=float(inc_angle)
-        scale_radius=float(scale_radius)
-        scale_height=float(scale_height)
-        pos_angle=float(pos_angle)
-        print(flux, inc_angle, scale_radius, scale_height, pos_angle)
-
-        # Now make a test image
-        test_profile = galsim.InclinedExponential(inc_angle*galsim.radians, scale_radius,
-                                                  scale_height, flux=flux)
-
-        check_basic(test_profile, "InclinedExponential")
-
-        # Check that h/r is properly given by the method and property for it
-        np.testing.assert_almost_equal(test_profile.scale_height/test_profile.scale_radius,
-                                       test_profile.scale_h_over_r)
-        np.testing.assert_almost_equal(test_profile.getScaleHeight()/test_profile.getScaleRadius(),
-                                       test_profile.getScaleHOverR())
-
-        # Rotate it by the position angle
-        test_profile = test_profile.rotate(pos_angle*galsim.radians)
-
-        # Check that the k value for (0,0) is the flux
-        np.testing.assert_almost_equal(test_profile.kValue(kx=0.,ky=0.),flux)
-
-        # Check that the drawn flux for a large image is indeed the flux
-        test_image = galsim.Image(5*image_nx,5*image_ny,scale=1.0)
-        test_profile.drawImage(test_image)
-        test_flux = test_image.array.sum()
-        np.testing.assert_almost_equal(test_flux,flux,decimal=3)
-
-        # Check that the centroid is (0,0)
-        centroid = test_profile.centroid()
-        np.testing.assert_equal(centroid.x, 0.)
-        np.testing.assert_equal(centroid.y, 0.)
-
-        # Check maxSB
-        # We don't do a great job at estimating this, but it should be in the right ball park,
-        # and typically too large.
-        test_profile.drawImage(test_image, method='sb', use_true_center=False)
-        print('max pixel: ',test_image.array.max(),' cf.',test_profile.maxSB())
-        np.testing.assert_allclose(test_image.array.max(), test_profile.maxSB(), rtol=0.3)
-        np.testing.assert_array_less(test_image.array.max(), test_profile.maxSB())
+    """ Performs various sanity checks on a set of InclinedExponential and InclinedSersic profiles. """
+    
+    for mode in ("InclinedExponential","InclinedSersic"):
+
+        print('flux, inc_angle, scale_radius, scale_height, pos_angle')
+        for flux, inc_angle, scale_radius, scale_height, pos_angle in zip(fluxes,
+                                                                          image_inc_angles,
+                                                                          image_scale_radii,
+                                                                          image_scale_heights,
+                                                                          image_pos_angles):
```

Comment:
Should also test the sersic options for InclinedSersic.  Otherwise, you aren't getting any sanity tests for n!=1, trunc!=0.